packages/components/stories/List.js

```diff
@@ -432,6 +450,31 @@ storiesOf('List', module)
 			</div>
 		);
 	})
+	.add('Virtualized - row click', () => {
+		const selectedItemsProps = cloneDeep(props);
+		const rowClickAction = action('onRowClick');
+		selectedItemsProps.list.itemProps.isActive = item => item.id === 0;
+		selectedItemsProps.list.itemProps.isSelected = item => item.id === 2;
+		selectedItemsProps.list.itemProps.onToggle = () => {};
+		selectedItemsProps.list.itemProps.onRowClick = event => rowClickAction('onRowClick', event.rowData);
+		return (
+			<div style={{ height: '60vh' }} className="virtualized-list" >
+				<h1>List</h1>
+				<p>
+					You can manage selection by passing 2 props : onRowClick and isActive.<br />
+					<b>onRowClick(event)</b> : item selection callback
+					<b>isActive(item)</b> : returns true if the item is selected
+					<pre>
+						listProps.itemProps.onRowClick = (event) => myRowClickCallback(event.rowData);<br />
+						listProps.itemProps.isActive = (item) => item.id === 0;<br />
+						&lt;List ... list=&#123;listProps&#125; &gt;<br />
```

Comment:
Nice :), thank you for the description
Don't forget to change it when you fixed the onRowClick arguments